.github/workflows/ci.yml

```diff
@@ -364,7 +356,7 @@ jobs:
         uses: actions/cache@1bd1e32a3bdc45362d1e726936510720a7c30a57
         with:
           path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('akka-javasdk-maven/pom.xml') }}
+          key: ${{ runner.os }}-maven-${{ hashFiles('akka-javasdk-parent/pom.xml') }}
```

Comment:
I think normally you'd key it with all pom files of the samples, since they could contribute to things that end up in the cache, but I think this is fine. Most important things to cache is anyway all the sdk deps.